npm/ng-packs/packages/schematics/src/utils/service.ts

```diff
@@ -93,7 +99,41 @@ export function createActionToSignatureMapper() {
 }
 
 function getMethodNameFromAction(action: Action): string {
-  return action.uniqueName.split('Async')[0];
+  return action.uniqueName.replace('Async', '');
```

Comment:
maybe it should be    return action.uniqueName.replace(new RegExp('Async$'), '');  (replace Async at end of line)